### ISSUE TITLE
[Bug]: ut TestMultiClusterCanWork yields `FATAL log-service taskservice/task_service.go:139 query task by primary key, return 0 records`

### ISSUE THREAD

**Issue by daviszhen** [matrixorigin/matrixone#19018]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Branch Name

daviszhen:0924-fix-dashboard

### Commit ID

lastest

### Other Environment Information

```Markdown
- Hardware parameters:
- OS type:
- Others:


https://github.com/matrixorigin/matrixone/actions/runs/11036054661/job/30653580813?pr=18973
```

### Actual Behavior

```
2024-09-25T15:31:28.4972754Z 2024/09/25 15:25:03.749600 +0000 INFO disttae/logtail_consumer.go:382 [logtail-consumer] subscribe tbl[db: 272389, tbl: 272390, sys_async_task] succeed
2024-09-25T15:31:28.4975665Z 2024/09/25 15:25:03.750984 +0000 INFO cn-service.lockservice lockservice/lock_table_keeper.go:82 keep lock table bind task {"service": "2-cn-2", "uuid": "2-cn-2", "cost": "34.772833823s"}
2024-09-25T15:31:28.4979145Z 2024/09/25 15:25:03.750957 +0000 FATAL log-service taskservice/task_service.go:139 query task by primary key, return 0 records {"service": "d623021c-8dd5-4c0f-a0c9-bdb16a560be8", "uuid": "d623021c-8dd5-4c0f-a0c9-bdb16a560be8"}
2024-09-25T15:31:28.4981422Z github.com/matrixorigin/matrixone/pkg/taskservice.(*taskService).Allocate
2024-09-25T15:31:28.4982725Z 	/home/runner/work/matrixone/matrixone/pkg/taskservice/task_service.go:139
2024-09-25T15:31:28.4983910Z github.com/matrixorigin/matrixone/pkg/hakeeper/task.allocateTask
2024-09-25T15:31:28.4985300Z 	/home/runner/work/matrixone/matrixone/pkg/hakeeper/task/task_scheduler.go:155
2024-09-25T15:31:28.4986642Z github.com/matrixorigin/matrixone/pkg/hakeeper/task.(*scheduler).allocateTasks
2024-09-25T15:31:28.4988000Z 	/home/runner/work/matrixone/matrixone/pkg/hakeeper/task/task_scheduler.go:119
2024-09-25T15:31:28.4989272Z github.com/matrixorigin/matrixone/pkg/hakeeper/task.(*scheduler).Schedule
2024-09-25T15:31:28.4990736Z 	/home/runner/work/matrixone/matrixone/pkg/hakeeper/task/task_scheduler.go:74
2024-09-25T15:31:28.4991973Z github.com/matrixorigin/matrixone/pkg/logservice.(*store).taskSchedule
2024-09-25T15:31:28.4993393Z 	/home/runner/work/matrixone/matrixone/pkg/logservice/store_hakeeper_check.go:227
2024-09-25T15:31:28.4994775Z github.com/matrixorigin/matrixone/pkg/logservice.(*store).tickerForTaskSchedule
2024-09-25T15:31:28.4996181Z 	/home/runner/work/matrixone/matrixone/pkg/logservice/store.go:927
2024-09-25T15:31:28.4997196Z FAIL	github.com/matrixorigin/matrixone/pkg/embed	68.117s
2024-09-25T15:31:39.9941398Z ?   	github.com/matrixorigin/matrixone/pkg/stream/adapter/kafka/test/proto/test_v1	[no test files]

```

### Expected Behavior

no failure

### Steps to Reproduce

```Markdown
https://github.com/matrixorigin/matrixone/actions/runs/11036054661/job/30653580813?pr=18973
```

**Comment by zhangxu19830126** [comment]
https://github.com/matrixorigin/matrixone/actions/runs/11099487248/attempts/1?pr=19063

**Comment by w-zr** [comment]
这个issue是通过主键查不到数据的问题。可能是workspace的问题。

**Comment by daviszhen** [comment]
https://github.com/matrixorigin/matrixone/actions/runs/11271253040/job/31343711628?pr=19200

**Comment by triump2020** [comment]
Pls ref to #19202

**Comment by triump2020** [comment]
Is it related to #19357  ?

**Comment by triump2020** [comment]
No repro yet recently!

**Comment by triump2020** [comment]
May be fixed by #19530
